pkg/graph/resolvers/mto_solution.go

```diff
@@ -214,3 +241,59 @@ func MTOSolutionDelete(ctx context.Context, logger *zap.Logger, principal authen
 		return nil
 	})
 }
+
+// MTOSolutionUpdateLinkedMilestonesWithTX updates the linked milestones for a solution, deleting ones that are not included in the list
+func MTOSolutionUpdateLinkedMilestonesWithTX(
```

Comment:
@OddTomBrooks, I think these new methods aren't needed. The work you did for the create common milestone should suffice here. I don't think these should / are doing anything different. Can you unify these please? (`MTOSolutionLinkMilestonesWithTX` is the matching method)